Implemented responsive FollowUs/Instagram section with IconButton chevrons

frontend010 opened the pull request:
- [ ] Made FollowUs/Instagram component fully responsive across all breakpoints
- [ ] Chevron buttons with IconButton from Shared/ui
- [ ] Adjusted image sizes and spacing to match Figma designs
- [ ] Ensured FOLLOW US button is centered and responsive
- [ ] Fixed mobile slider behavior with proper prev/next navigation

frontend010 committed:
feat: implement responsive Instagram section with IconButton chevrons

erianiel commented on .vscode/settings.json:
`.vscode/settings.json` is meant to be shared across the team so we all use the same formatter and formatting rules.

Switching the default formatter from Prettier to the VS Code language features changes formatting behavior for everyone, so this probably shouldn't be committed unless we've agreed on it as a team. 

I'd suggest reverting this and keeping Prettier as the default formatter in the repo settings. Personal formatter preferences can stay in local vs code user settig instead. Thanks!

erianiel replied:
@frontend010 This file should not be changed, please revert it.

erianiel commented on src/components/FollowUs/FollowUs.module.scss:
```diff
@@ -1,19 +1,27 @@
 
 .container {
-  width: 1152px;
+  width: 1150px;
```
Just a small thought on the width here: since this component needs to align with others on the same page, it might be better to keep its width consistent with the layout. Instead of 1150px, we could use width: 100%, which means already width: 1152px, relying on the layout constraint.

erianiel commented on src/components/FollowUs/FollowUs.module.scss:
```diff
@@ -1,19 +1,27 @@
+  @media (max-width: $tablet) {
+    width: 100%;
+    flex-direction: column;
+    align-items: flex-start;
```
Maybe we shouldn't write again the `align-items: flex-start;` since it is already declared in line 17.

erianiel commented on src/components/FollowUs/FollowUs.module.scss:
```diff
@@ -25,8 +33,23 @@
   line-height: 32px;
```
In mobile version, according to Figma the font-size is 20px and the line-height is 26px.

If possible, could we use rem units instead of px for better responsiveness? The codebase still contains some px values, but we’re gradually replacing them with rem to improve scalability and consistency across devices.

erianiel commented on src/components/FollowUs/FollowUs.module.scss:
```diff
@@ -74,23 +117,39 @@
-  transition:
-    background 0.2s ease,
-    color 0.2s ease;
+  transition: background 0.2s ease, color 0.2s ease;
```
This `.button` class seems to duplicate several properties that are already defined in the reusable Button component.
Could we refactor it to avoid duplication and keep only what’s strictly needed here?

erianiel reviewed (changes requested):
The title and cards appear broken across viewports and don't match the expected layout which blocks the review. It would be good to take a look and to fix these issues first, before continuing the review. 

I added some comments, I hope they could help a little bit.

frontend010 committed:
FollowUs reviews updated

frontend010 committed:
follow-us-instragram reviews updated

frontend010 commented:
- Reverted .vscode/settings.json formatter change (kept Prettier)
- Removed hard-coded container widths and aligned layout to width: 100%
- Cleaned up duplicate CSS declarations
- Updated mobile typography to match Figma and switched to rem
- Refactored Follow Us CTA to use shared Button component
- Removed duplicated button styles and matched hover behavior to Figma

erianiel commented on src/components/Recommendations/Recommendations.module.scss:
Please revert these changes, they are related to the settings.json file changes.

erianiel commented on src/components/FollowUs/FollowUs.module.scss:
- At the `tablet` and `smallScreen` breakpoints, it looks like the section title is still displayed side by side with the component. 
At those breakpoints, it should stack on top (similar to Reviews / Recommendation section behave). 
It is also causing the component to overflow the layout


<img width="618" height="767" alt="Small screen breakpoint: section title displayed side by side instead of stacked above the reviews content" src="https://github.com/user-attachments/assets/98e45c19-13e3-4727-a438-eda13896323f" />
 #### smallscreen breakpoint. 

-------

<img width="662" height="557" alt="Image" src="https://github.com/user-attachments/assets/9125f874-093d-471f-9722-769c2c7ad2c2" />
 #### tablet breakpoint. 

-------

- Just a few quick remainders:

    -  we generally prefer using `rem` instead of px. It improves the accessibility since `rem` scales with the user's base font-size.
Could you, please, convert the modified pixels values to rem where possible?
For instance 16px = 1rem / 24px = 1.5rem / 345px = 21.5rem

   - by convention we follow a mobile-first approach, so we usually rely on `min-width` media queries. This helps keep components easier to maintain and evolve over time.
I know that the codebase can be a bit inconsistent and some files mixes them, but our goal is to gradually improve it and make it solid. So when making changes, please, it would be great to stick to `min-width` only.
Also, mixing min-width and max-width in the same rule can sometimes lead to unexpected CSS conflicts, so we try to avoid patterns like this:

```css 

@media (min-width: $tablet) and (max-width: $smallScreen) {
  width: 220px;
  height: 312px;
}

erianiel commented on src/components/FollowUs/FollowUs.module.scss:
```diff
@@ -35,62 +61,78 @@
+    width: 100%;
+    margin-left: 0;
+    display: grid;
+    gap: 24px;
```
Could you double-check the gap on Figma, please?
On the mobile breakpoint it looks like 16px (1rem) to me, but I might be missing something.
